README.md

```diff
@@ -272,19 +272,18 @@ tools/pdf2image.sh en 5
 If you'd like to show a single image of two instances of the same page side-by-side (before|after style), you can use the following script:
 
 ```bash
-tools/compare_pages.sh <FILE_COMPARED_AGAINST> <LANGUAGE> <FIRST_PAGE> <LAST_PAGE>
+tools/compare_pages.sh
 ```
 
-Let's assume you have `main_en.pdf` in your home directory downloaded from GitHub, and in your current working directory you have a build you're working on.
-You'd like to have images comparing pages 38 through 41.
-Here's how to use it:
+The script takes local `main_<language>.pdf` that you built and which contains your changes and compares it with the latest build
+of the same language in this repository (e.i. the baseline). 
+
+Run the following for further explanation and usage:
 
 ```bash
-tools/compare_pages.sh ~/main_en.pdf en 38 41
+tools/compare_pages.sh  --help
```

Comment:
Don't do that, please leave at least one working example with an expected outcome. You can mention help below it.

Reply:
Working examples are in the help with explanations. I feel it's better to keep them with code than uncoupled in readme

Reply:
No, it's not. Please leave at least one thing that can be easily copy-pasted. It's tempting just to write RTFM, but it's a lot nicer to have an actual working example.

Reply:
I don't agree with you on this as I've seen too many outdated readmes. IMHO it's always better to keep all documentation in one place, once it gets spread to multiple places, chances are somebody forgets to update them all. 

But I'll keep it for the sake of consitency with the rest of the readme.